mesonbuild/interpreter/type_checking.py

```diff
@@ -892,4 +892,5 @@ def _dep_fallback_feat_validator(val: T.List[str]) -> T.Iterable[FeatureCheckBas
         since='0.40.0',
         validator=in_set_validator({e.value for e in DependencyMethods}),
     ),
+    KwargInfo('modules', ContainerTypeInfo(list, str), default=[], listify=True, since='0.54.0'),
```

Comment:
Any feature new to remove somewhere?

Reply:
I couldn't find any, I think this may have simply never been annotated